pkg/splunk/enterprise/monitoringconsole.go

```diff
@@ -198,6 +207,50 @@ func getMonitoringConsoleStatefulSet(client splcommon.ControllerClient, cr splco
 		},
 	}
 
+	// update template to include storage for etc and var volumes
+	if spec.EphemeralStorage {
```

Comment:
You mean separate CR for MC?

Reply:
My bad, yes, MC.

Reply:
Since we have a separate STS for MC, I think we can just go with PVCs, so that, we don't need to worry about the combinations of storages across the CRs here.